frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/pushdialog/GitPushDialog.java

```diff
@@ -65,38 +63,32 @@ public GitPushDialog(
     this.isForcePushRequired = isForcePushRequired;
     this.project = project;
 
-    @SuppressWarnings("nullness:assignment.type.incompatible") @Initialized MyVcsPushDialog dialog = new MyVcsPushDialog(this,
-        project, selectedRepositories.asJava(), pushSource);
+    MyVcsPushDialog dialog = new MyVcsPushDialog(project, selectedRepositories.asJava(), pushSource);
```

Comment:
Note that this line (instantiation of inner classes from c'tor before enclosing object is fully initialized) only passes NullnessChecker because of https://github.com/typetools/checker-framework/issues/3407

Reply:
So... let us pray that this time it can't NPE in the runtime